packages/grid/test/drag-and-drop-polymer.test.js

```diff
@@ -0,0 +1,2 @@
+import '../all-imports.js';
```

Comment:
The `drag-and-drop.test.js` only imports `vaadin-grid.js`, no need to use `all-imports` here.

Reply:
Done